packages/block-editor/src/hooks/layout.scss

```diff
@@ -10,6 +10,10 @@
 			margin: auto 0 $grid-unit-05 $grid-unit-10;
 		}
 	}
+
+	.components-input-control {
```

Comment:
For robustness, I would recommend passing your own classNames to the UnitControls (`.block-editor-hooks__something`), or scoping it to `.components-unit-control` at the very least since it isn't a given that UnitControls will always have a `.components-input-control` class under the hood.